integration-tests/conftest.py

```diff
@@ -1,10 +1,27 @@
 import pytest
+import os
+import subprocess
 
 
+@pytest.fixture(scope="session", autouse=True)
+def install_katello_rpm(test_config):
+    if "satellite" in os.getenv('ENV_FOR_DYNACONF'):
+        # install katello rpm before register system against Satellite
+        satellite_hostname = test_config.get("candlepin", "host")
+        cmd = 'rpm -qa | grep katello | xargs rpm -e;rpm -Uvh http://%s/pub/katello-ca-consumer-latest.noarch.rpm' %satellite_hostname
```

Comment:
> rpm -qa | grep katello | xargs rpm -e

This can be improved a bit:
- `rpm -q` accepts wildcards, so you can do `rpm -qa 'katello*'` and drop the `grep`
- `xargs` complains in case there are no inputs, which is the case of no katello packages already installed; please pass `--no-run-if-empty` as argument to `xargs` so it does not fail in that case